core/templates/services/navigation.service.spec.ts

```diff
@@ -38,7 +38,10 @@ describe('Navigation Service', () => {
   beforeEach(angular.mock.inject(function($injector) {
     NavigationService = $injector.get('NavigationService');
 
-    // @ts-ignore
+    // There is a typescript error here because the type of element doesn't
```

Comment:
Too vague. In what way do they not match? Also put quotes around "element" if you are referring to that term in the codebase and not a generic element.

Reply:
done